pkg/kubelet/kubelet_node_status_test.go

```diff
@@ -1700,6 +1707,216 @@ func TestUpdateDefaultLabels(t *testing.T) {
 	}
 }
 
+func TestUpdateDefaultResources(t *testing.T) {
+	cases := []struct {
+		name         string
+		initialNode  *v1.Node
+		existingNode *v1.Node
+		expectedNode *v1.Node
+		needsUpdate  bool
+	}{
+		{
+			name: "no update needed when capacity and available of the existing node are not nil",
```

Comment:
nit s/available/allocatable

Reply:
Thanks a lot @bobbypage ! The nit has been fixed.